.github/workflows/release.yml

```diff
@@ -146,6 +146,48 @@ jobs:
           name: wheels
           path: dist
 
+  linux-eol-wheels:
+    # starting from ubuntu-22.04 Python 3.6 isn't available anymore
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [x86_64]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: |
+            3.6
+      - name: Patch Cargo.toml
+        run: |
+          # since we're using an older maturin version, metadata inheritance doesn't work -
+          # overwrite it explicitly
+          sed -i \
+              -e "s/categories.workspace = true/$(grep 'categories' Cargo.toml)/" \
+              -e "s/edition.workspace = true/$(grep 'edition' Cargo.toml)/" \
+              -e "s/keywords.workspace = true/$(grep 'keywords' Cargo.toml)/" \
+              -e "s/license.workspace = true/$(grep 'license' Cargo.toml)/" \
+              -e "s/repository.workspace = true/$(grep 'repository' Cargo.toml)/" \
+              -e "s/rust-version.workspace = true/$(grep 'rust-version' Cargo.toml)/" \
+              -e "s/version.workspace = true/$(grep 'version' Cargo.toml)/" \
+              pineappl_py/Cargo.toml
```

Comment:
```suggestion
          sed -i \
              -e "s@categories.workspace = true@$(grep 'categories' Cargo.toml)@" \
              -e "s@edition.workspace = true@$(grep 'edition' Cargo.toml)@" \
              -e "s@keywords.workspace = true@$(grep 'keywords' Cargo.toml)@" \
              -e "s@license.workspace = true@$(grep 'license' Cargo.toml)@" \
              -e "s@repository.workspace = true@$(grep 'repository' Cargo.toml)@" \
              -e "s@rust-version.workspace = true@$(grep 'rust-version' Cargo.toml)@" \
              -e "s@version.workspace = true@$(grep 'version' Cargo.toml)@" \
              pineappl_py@Cargo.toml
```

Reply:
Fixed in commit 2d632d3b440c58e4909c8a1518ba07e7e986ffc9.